.github/workflows/ansible-lint.yml

```diff
@@ -1,11 +1,15 @@
-name: Run ansible-lint on the playbooks and roles
+# Run Ansible lint in playbooks and roles
+name: Run ansible-lint
 
 # Controls when the action will run.
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   pull_request:
     branches: [main]
-
+  push:
+    branches: [main]
```

Comment:
The idea here is to run the workflow after every time there is a commit (PR merged) in the main branch, you will find out how many times things break and you notice the glitch in the merge commit. But it is indeed not required.

Reply:
> you will find out how many times things break and you notice the glitch in the merge commit.

hum, I took time to understand what you meant, but now I get it :) *glitch in the merge commit*, so when the `main` and the PR are joined, instead of considering the PR as a standalone branch.

I see 🤔 

Just FYI, OpenShift CI (ie, what's [there](https://github.com/openshift/release/tree/fd3b3af544003777024c74db9a8f32cb556b4da1/ci-operator/config/openshift-psap/topsail)) takes care of that by always testing the merge commit, instead of only the PR HEAD.

and I _think_ that enforcing a linear history to merge the PR also takes care of that point, as it enforces that the merge commit is empty.

but anyway, we can keep the hook, it makes sense in a general case